INSTALL

```diff
@@ -157,10 +157,13 @@
                    "--cross-compile-prefix=mipsel-linux-gnu-" suffices
                    in such case. Needless to mention that you have to
                    invoke ./Configure, not ./config, and pass your target
-                   name explicitly.
+                   name explicitly. Even keep in mind that --openssldir
+                   refers to target's file system, not one you are running
+                   Configure on.
 
   --debug
-                   Build OpenSSL with debugging symbols.
+                   Build OpenSSL with debugging symbols and optimizations
+                   off.
```

Comment:
maybe add "turned"? It would probably also be more clear to add "on" after debugging symbols, now it reads as both off.